src/Contract/Time.purs

```diff
@@ -13,15 +16,12 @@ module Contract.Time
 
 import Prelude
 
-import Contract.Chain
-  ( BlockHeaderHash(BlockHeaderHash)
-  , ChainTip(ChainTip)
-  , Tip(Tip, TipAtGenesis)
-  , getTip
-  ) as Chain
-import Contract.Monad (Contract, liftedE)
+import Contract.Chain (getTip) as TipChain
```

Comment:
We should not re-export the same identifier more than once from `Contract.*` modules. The reason is, language-server worflow for adding imports gets easier when there is only one alternative (there is no need to pick a module). Please remove this re-export.